concepts/05 UI Components/FileUploader/50 Customize FileUploader.md

```diff
@@ -0,0 +1,162 @@
+This tutorial demonstrates how to customize FileUploader appearance.
+
+<div class="simulator-desktop-container" data-view="/Content/Applications/25_1/UIWidgets/FileUploader/Customization/index.html, /Content/Applications/25_1/UIWidgets/FileUploader/Customization/index.js, /Content/Applications/25_1/UIWidgets/FileUploader/Customization/index.css"></div>
+
+Specify the [dropZone](/Documentation/ApiReference/UI_Components/dxFileUploader/Configuration/#dropZone) property to integrate a custom drop zone. Use the `width` and `height` CSS styles to modify the dimensions of the dropzone. This tutorial also implements the `background-color` and `border-radius` styles:
+
+---
+
+##### jQuery
+
+    <!-- tab: index.html -->
+    <div class="file-uploader-block"></div>
+    <div id="file-uploader"></div>
+
+    <!-- tab: index.js -->
+    $(() => {
+        $('#file-uploader').dxFileUploader({
+            dropZone: '.file-uploader-block',
+        })
+    })
+
+    <!-- tab: styles.css -->
+    .file-uploader-block {
+        width: 350px;
+        height: 200px;
+        background-color: #f5f5f5;
+        border-radius: 16px;
+    }
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <div class="file-uploader-block"></div>
+    <dx-file-uploader dropZone=".file-uploader-block"></dx-file-uploader>
+
+    <!-- tab: app.component.css -->
+    .file-uploader-block {
+        width: 350px;
+        height: 200px;
+        background-color: #f5f5f5;
+        border-radius: 16px;
+    }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <div class="file-uploader-block"></div>
+        <DxFileUploader drop-zone=".file-uploader-block" />
+    </template>
+
+    <!-- tab: styles.css -->
+    .file-uploader-block {
+        width: 350px;
+        height: 200px;
+        background-color: #f5f5f5;
+        border-radius: 16px;
+    }
+
+##### React
+
+    <!-- tab: App.tsx -->
+    export default function App() {
+        return (
+            <div class="file-uploader-block"></div>
+            <FileUploader dropZone=".file-uploader-block" />
+        )
+    }
+
+    <!-- tab: styles.css -->
+    .file-uploader-block {
+        width: 350px;
+        height: 200px;
+        background-color: #f5f5f5;
+        border-radius: 16px;
+    }
+
+---
+
+This tutorial places FileUploader inside the custom drop zone container:
```

Comment:
This is how I think about this sentence. 

We already applied some customizations to a FileUploader component (instructions and code above). So now it's "the FileUploader". 

But "custom drop zone container" is something new that we haven't worked with before. So I would refer to it as "a custom drop zone container" (this first time we mention it).

```suggestion
This tutorial places the FileUploader inside a custom drop zone container:
```